### PR TITLE
docs: Improve "pre-commit" description

### PR DESCRIPTION
**Pull request by richb-hanover**
I thought further about how to explain `pre-commit`. This wording is better.

### CODE REVIEW

File: USING_DOCKER.md
Comment (max-sixty) [inline USING_DOCKER.md]: I _think_ this may just install the plugins and `pre-commit install` is required to add the git automation, so it runs on each commit; is that valid at all?

(it might also be worth linking to the pre-commit docs; we won't explain it better than they do!)

File: USING_DOCKER.md
Comment (max-sixty) [inline USING_DOCKER.md]: I do think it's worth linking to https://github.com/prql/prql/blob/main/DEVELOPMENT.md#tests here; we give an explanation of what they do and how they fit into the overall testing approach. Or do you think that's kinda confusing? Please criticize away for that doc!

**Review by max-sixty — APPROVED** [review]
Great, I do think that's better, thanks for coming back to it.

I gave you merge permissions, so feel free to merge when you're ready. Or @ me if you want me to take another look.

**Comment by richb-hanover** [comment]
Good points. I'll look at it this weekend.

File: USING_DOCKER.md
Comment (richb-hanover) [reply USING_DOCKER.md]: You're right - we need `pre-commit install --install-hooks`. 

The Static Checks section of DEVELOPMENT.md already links to the pre-commit docs.

File: USING_DOCKER.md
Comment (richb-hanover) [reply USING_DOCKER.md]: Definitely. DRY (Don't Repeat Yourself)

**Comment by max-sixty** [comment]
Nice, thanks @richb-hanover !

**Comment by richb-hanover** [comment]
> I gave you merge permissions, so feel free to merge when you're ready. Or @ me if you want me to take another look.

@max-sixty I am honored that you gave me merge permissions, but I am getting distracted by all the messages I now receive. 

I am going to ask you to revoke it: I'll continue to follow `prql` closely (especially since I have `prql-query` to gnaw on) but on a less intense basis. Thanks.

File: USING_DOCKER.md
Comment (max-sixty) [inline USING_DOCKER.md]: I thin this needs to be `--install-hooks`; the current command raises for me

(I can't seem to make a suggestion in GH after merge)

**Comment by max-sixty** [comment]
Thanks for the PR!

> @max-sixty I am honored that you gave me merge permissions, but I am getting distracted by all the messages I now receive.

Of course, sorry for the distractions. FWIW you can unsubscribe from them with the `Unwatch` button at the top left. Let me know if that works; if I don't hear from you I'll do as you asked!

image

**Comment by richb-hanover** [comment]
Yes - I fixed the `--install-hooks` in the final push (I'm not sure why it showed up without the `--` above.

And I did unwatch - let's see if that helps. (It seemed that three of every two messages that came to my Inbox were from Github :-)

**Comment by max-sixty** [comment]
OK — adding you back!

File: USING_DOCKER.md
Comment (max-sixty) [reply USING_DOCKER.md]: Weird, as you say, the real code is correct

**Comment by richb-hanover** [comment]
> OK — adding you back!

Thank you, but I'm gonna decline the invitation. I have a ton on my plate - much of it not "computer related" (I'm setting up condominium (!) so I shouldn't be spending to much time on this.) I'm happy to keep playing with prql and throwing comments over the wall... Thanks.

**Comment by max-sixty** [comment]
No prob, thanks for your contributions so far and any more will be welcome whenever they come!

**Pull request merged**
